GooglePlayInstant/Editor/PlayInstantBuilder.cs

```diff
@@ -204,7 +204,7 @@ public static bool DisplayBuildErrorDialog(string message)
         public static void LogError(string message)
         {
             Debug.LogErrorFormat("Build error: {0}", message);
-            EditorUtility.DisplayDialog(BuildErrorTitle, message, OkButtonText);
+            //EditorUtility.DisplayDialog(BuildErrorTitle, message, OkButtonText);
```

Comment:
It looks like we can do this to check for headless mode
```
private static bool IsHeadlessBuild()
{
  return SystemInfo.graphicsDeviceType == UnityEngine.Rendering.GraphicsDeviceType.Null;
}

public static void LogError(string message)
{
  if (IsHeadlessBuild())
  {
    throw new Exception...
  }
  else
  {
    Debug.LogErrorFormat("Build error: {0}", message);
    EditorUtility.DisplayDialog(BuildErrorTitle, message, OkButtonText);
  }
}
```

Reply:
Oh I like that. I'll make this a permanent change then.